ksql-common/src/main/java/io/confluent/ksql/function/UdfFactory.java

```diff
@@ -48,16 +50,15 @@ public UdfFactory copy() {
   }
 
   void addFunction(final KsqlFunction ksqlFunction) {
-    final List<Schema.Type> paramTypes = ksqlFunction.getArguments()
-        .stream()
-        .map(Schema::type).collect(Collectors.toList());
+    final List<FunctionParameter> paramTypes
+        = mapToFunctionParameter(ksqlFunction.getArguments(), false);
```

Comment:
Why the 'false' into this function, which is saying 'ignore nulls'....?  Surely a `KsqlFunction` shouldn't have null parameter types. Check this in KsqlFunction constructor, not in this class.